tools/topology/sof-tgl-sdw-max98373-rt5682.m4

```diff
@@ -56,9 +56,12 @@ define(`SMART_BE_ID', 2)
 # Playback related
 define(`SMART_PB_PPL_ID', 3)
 define(`SMART_PB_CH_NUM', 2)
-define(`SMART_TX_CHANNELS', 4)
-define(`SMART_RX_CHANNELS', 8)
-define(`SMART_FB_CHANNELS', 8)
+# channel number for playback on sdw link
+define(`SMART_TX_CHANNELS', 2)
+# channel number for capture on sdw link
+define(`SMART_RX_CHANNELS', 4)
+# channel number generated by smart amplifier based on capture stream
```

Comment:
and this even less clear, sorry.

Reply:
@plbossart  copy the comment in sof-smart-amplifier.m4

Reply:
FB_CHANNELS is used in mux input/output configuration, please check pipe-amp-ref-capture.m4. I spent almost an hour to check the FW but I didn't find anything about it. Then I consulted to smart-amp expert. The FW patch is still not merged in master !  It is only adopted in release branch.  According to tgl-006-drop-stable, the FB channel is the number of capture stream (I/V) sent to user mode application.